src/milind/Exam1/UserNameValidation.java

```diff
@@ -0,0 +1,40 @@
+package milind.Exam1;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class UserNameValidation {
+	
+	WebDriver driver;
+	void verifyUserName() {
+		driver = PredefinedActions.start();
+		driver.findElement(By.xpath("//a[@id='demotable']")).click();
+		int rowCount = driver.findElements(By.xpath("//table[@id='table1']/tbody/tr")).size();
+		List<Character> firstNameFirstLetter = new ArrayList<Character>();
+		List<String> lastFullName = new ArrayList<String>();
+		for(int index = 1; index <= rowCount; index++) {
+			firstNameFirstLetter.add(driver.findElement(By.xpath("//table[@id='table1']/tbody/tr["+index+"]/td[2]")).getText().charAt(0));
+			lastFullName.add(driver.findElement(By.xpath("//table[@id='table1']/tbody/tr["+index+"]/td[3]")).getText().toLowerCase());
+		}
+		for(int index = 0; index < rowCount; index++) {

```

Comment:
you can compare data with single loop only, not require to add into List and then compare username, interview perspective this will give negative impact.